### PR TITLE
Release v1.0.0: Out of beta

### PR DESCRIPTION
**Pull request by jidicula**
# New Features
* Metro outages
* role functionality with `?inrole` to list all users in a role, and `?roles` to list all roles in a server.
* better windchill calculation in the weather function
* UTF-8 support for LaTeX commands
* dependency updates
* new additions to gitignore for .ini files

# Refactors
* new subscriptions cog for functions that run on a schedule rather than by a user command

### CODE REVIEW

**Comment by dbenjaminmiller** [comment]
Isn't `?inrole` for listing the users in a role, and `?roles` for listing roles in the server?

**Comment by jidicula** [comment]
> Isn't `?inrole` for listing the users in a role, and `?roles` for listing roles in the server?

Correct, I've updated the PR description accordingly, thanks!

**Comment by le-potate** [comment]
Out of beta 🎉 Good job everyone

File: Main.py [hunk 1]
```diff
@@ -38,8 +38,16 @@
...
-    "cogs.reminder", "cogs.memes", "cogs.helpers", "cogs.mod", "cogs.score",
-    "cogs.quotes", "cogs.images", "cogs.currency"
+    "cogs.currency",
+    "cogs.helpers",
+    "cogs.images",
+    "cogs.info",
+    "cogs.memes",
+    "cogs.mod",
+    "cogs.quotes",
+    "cogs.reminder",
```
Comment (dbenjaminmiller) [inline Main.py]: What's this extra line of space for

**Pull request merged**